intelmq_manager/static/js/configs.js

```diff
@@ -373,6 +375,9 @@ function fill_bot(id, group, name) {
         insertKeyValue(key, bot.defaults[key], BORDER_TYPES.DEFAULT, false);
     }
 
+    var periodpos = bot['module'].lastIndexOf('.');
```

Comment:
Might be `const` instead of `var`. I know I know the code is full of `var` but that's an old way.